packages/widgets/src/lib/utils/ViewUtils.ts

```diff
@@ -128,6 +128,37 @@ export function getPipelineTask(modelPipeline: PipelineType): PipelineType {
 	return modelPipeline === "text2text-generation" ? "text-generation" : modelPipeline;
 }
 
+interface RunShortcutParams {
+	isLoading: boolean;
+	isDisabled: boolean;
+	onClick: () => void;
+}
+
+/**
+ * Svelte action that will call inference endpoint when a user hits cmd+Enter on a current html element
+ */
+export function runShortCut(
```

Comment:
Maybe?
```suggestion
export function onCmdEnter(
```

Reply:
handled in 7a17f3824ed8a265064ba75fd146929796975618